.storybook/config.js

```diff
@@ -6,6 +6,10 @@ import { create } from '@storybook/theming';
 import { contexts } from './configs/contexts';
 
 addParameters({
+  readme: {
+    // You can set a code theme globally.
+    codeTheme: 'github',
```

Comment:
This was why we lost the theme

https://github.com/tuchk4/storybook-readme/issues/186